app/forms/curation_concerns/monograph_form.rb

```diff
@@ -3,5 +3,10 @@
 module CurationConcerns
   class MonographForm < CurationConcerns::Forms::WorkForm
     self.model_class = ::Monograph
+    self.terms += [:date]
+    self.terms += [:isbn]
+    self.terms += [:editor]
+    self.terms += [:copyright_holder]
+    self.terms += [:buy_URL]
```

Comment:
These don't have to be on separate lines.  You can put them all on one line like this:
`self.terms += [:date, :isbn, :editor, :copyright_holder, :buy_URL]`